### PR TITLE
Unify DeleteAction API

### PR DESCRIPTION
**Pull request by mmintel**
Hey there,

in order to fix https://github.com/tinacms/tinacms/issues/884 I've prepared a little PR for you. I hope this covers everything and solves it like intended.

I saw you are using conventional commits, I recommend to setup husky to check for that :) 

Cheers
Marc

### CODE REVIEW

**Comment by mmintel** [comment]
I guess those failing pipelines are not related to my actual code changes?

**Comment by ncphillips** [comment]
Amazing! Thank you 😄

Correct. Failing pipelines are because your fork doesn’t have the same secrets set. We will merge into another branch to verify the pipeline.

We’re already using husky to run prettier, so using it to check commit formats is a great idea. Thanks for that too 👍

File: packages/gatsby-tinacms-git/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "docs": "npx typedoc"
   },
   "dependencies": {
+    "tinacms": "^0.16.0",
```
Comment (ncphillips) [inline packages/gatsby-tinacms-git/package.json]: This should instead by a `peerDependency` and a `devDependency`. See [gatsby-tinacms-json](https://github.com/tinacms/tinacms/blob/master/packages/gatsby-tinacms-json/package.json#L34-L43) for an example.

File: packages/gatsby-tinacms-json/package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
     "tinacms": "^0.16.0"
   },
   "dependencies": {
+    "gatsby-tinacms-git": "^0.4.0",
```
Comment (ncphillips) [inline packages/gatsby-tinacms-json/package.json]: This should also be a `peerDependency`:

```
"gatsby-tinacms-git": ">=0.4",
```

File: packages/gatsby-tinacms-mdx/package.json [hunk 3]
```diff
@@ -36,6 +36,7 @@
     "tinacms": "^0.16.0"
   },
   "dependencies": {
+    "gatsby-tinacms-git": "^0.4.0",
```
Comment (ncphillips) [inline packages/gatsby-tinacms-mdx/package.json]: peer

File: packages/gatsby-tinacms-remark/package.json [hunk 4]
```diff
@@ -37,6 +37,7 @@
     "tinacms": "^0.16.0"
   },
   "dependencies": {
+    "gatsby-tinacms-git": "^0.4.0",
```
Comment (ncphillips) [inline packages/gatsby-tinacms-remark/package.json]: peer

**Comment by ncphillips** [comment]
Looks great! Just a `package-lock.json` conflict in the way :)

**Comment by ncphillips** [comment]
@all-contributors please add @mmintel for code

**Pull request merged**